cmd/lakectl/cmd/annotate.go

```diff
@@ -81,4 +94,6 @@ func splitOnNewLine(str string) string {
 //nolint:gochecknoinits
 func init() {
 	rootCmd.AddCommand(annotateCmd)
+
+	annotateCmd.Flags().BoolP("recursive", "r", false, "recursively list and annotate all entries under the given path")
```

Comment:
```suggestion
	annotateCmd.Flags().BoolP("recursive", "r", false, "recursively annotate all entries under a given path or prefix")
```
I think annotate is by definition also listing so it's kind of redundant in my opinion. As well I think it works for a prefix and not only for a path. Change at your discretion.